### ISSUE TITLE
0.4-SNAPSHOT immediately crashes after "downloading the minecraft jar"

### ISSUE THREAD

**Issue by QPCrummer** [PatchworkMC/patchwork-patcher#91]
I have been using the old patcher for a while (the 1.12 one). I recently started a 1.16.5 modded server, and wanted Tinker's Construct 1.16.4 version. So I got the latest patcher, V0.4 SNAPSHOT. I put Tinkers into the input folder, then double clicked on the patcher. It booted up, but then would immediately crash after it was done downloading the Minecraft jar. Thanks you guys so much for helping and continue changing Minecraft modding  forever! If you need anything, let me know.

**Comment by TheGlitch76** [comment]
1. There is no 1.12 version of Patchwork
2. Patchwork doesn't work with that mod, as stated in our discord
3. Please provide the crash log in a site like pastebin

**Comment by QPCrummer** [comment]
My bad, it was the 1.14 version of the patcher. Anyways, I have also tried Biomes o plenty. The same thing happens. Also I don't see any logs. I'll give you the exact steps I did to cause it to happen:

1. Downloaded a mod (biomes o plenty)
2. put that mod into the "input" folder
3. double clicked on the patchwork patcher jar file to open the ui
4. clicked patch
5. waited until it got to downloading minecraft jar. Then it took me back to the patch screen and I waited for about 20 minutes and nothing happened

I am running Java 15. I don't know if the type of computer I have really matters, but I'll give it to you anyways. I have a ryzen 7 5800x with a radeon rx 480. 64 GB of ram also. The location of the patcher was on my desktop in a folder.

The only other thing to note is that after clicking the patcher the first time, it made three folders, input, output, then data. Inside of data it downloaded forge universal 1.16.4.
